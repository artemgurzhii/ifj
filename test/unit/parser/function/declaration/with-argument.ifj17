declare function foo(bar as integer) as integer
