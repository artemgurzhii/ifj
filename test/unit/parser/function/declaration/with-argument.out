(declare
  (function foo -> (id integer)
    (decl
      (id bar)
      : (id integer))))
